src/components/Header/Header.styles.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
```

Comment:
The header doesn't match the designs perfectly yet

![image](https://user-images.githubusercontent.com/9722792/77818787-b37ed480-70d5-11ea-96ce-3f408ca53932.png)

Reply:
![Selection_007](https://user-images.githubusercontent.com/2856310/77827357-5cc3cb80-70d2-11ea-99d4-6477bfcdab01.jpg)
I made some padding tweaks but one (and only one) of the menu item is still 1px off

Reply:
I don't even see it :100: